src/SamplesApp/SamplesApp.UWP/SamplesApp.UWP.csproj

```diff
@@ -104,7 +104,7 @@
       <Version>6.1.0</Version>
     </PackageReference>
     <PackageReference Include="Microsoft.UI.Xaml">
-      <Version>2.5.0</Version>
+      <Version>2.6.0-prerelease.210113001</Version>
```

Comment:
Is this required?

Reply:
Yeah the [In]DeterminateSource Property is only available as of the 2.6.0 prerelease, otherwise the UWP Samples app won't build

Reply:
d'oh!